lms/templates/admin/macros.html.jinja2

```diff
@@ -44,6 +44,14 @@
 {% endmacro %}
 
 
+{% macro settings_textarea(object, label, setting, sub_setting, field_name, default='') %}
+    {% call field_body(label) %}
+        {% set text = object.settings.get(setting, sub_setting, default) or ''  %}
+        <textarea class="textarea" name="{{ setting }}.{{ sub_setting }}">{{ text }}</textarea>
```

Comment:
It might be nicer to call this "settings.{{ setting }}.{{ sub_setting }}". That way you could deal with all possible at once maybe?

Reply:
Makes sense but it will mean changing the settings over the application instance template, probably for a follow up PR.